pf4j/src/main/java/org/pf4j/processor/ExtensionAnnotationProcessor.java

```diff
@@ -229,4 +254,26 @@ private ExtensionStorage createStorage() {
         return storage;
     }
 
+    private static AnnotationMirror getAnnotationMirror(TypeElement typeElement, Class<?> clazz) {
+        // get annotation of a type element
+        // as described at https://stackoverflow.com/a/10167558
+        String clazzName = clazz.getName();
+        for (AnnotationMirror m : typeElement.getAnnotationMirrors()) {
+            if (m.getAnnotationType().toString().equals(clazzName)) {
+                return m;
+            }
+        }
+        return null;
+    }
+
+    private static AnnotationValue getAnnotationValue(AnnotationMirror annotationMirror, String key) {
```

Comment:
Move this method in https://github.com/pf4j/pf4j/blob/master/pf4j/src/main/java/org/pf4j/util/ClassUtils.java